### PR TITLE
add top to Dialog, #277

### PR DESCRIPTION
**Pull request by Leopoldthecoder**
Please makes sure these boxes are checked before submitting your PR, thank you!

* [ ] Make sure you follow Element's [Contributing Guide](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.md).
* [ ] Rebase before creating a PR to keep commit history clear.
* [ ] Add some descriptions and refer relative issues for you PR.

### CODE REVIEW

**Comment by Leopoldthecoder** [comment]
#277

File: packages/dialog/src/component.vue [hunk 1]
```diff
@@ -59,13 +63,12 @@
...
-    },
+      },
 
-    data() {
-      return {
-        dynamicTop: 0
-      };
+      top: {
+        type: String,
+        default: '15%'
```
Comment (furybean) [inline packages/dialog/src/component.vue]: 现在是 15% 么，设计就是这个样子么？

Comment (Leopoldthecoder) [inline packages/dialog/src/component.vue]: 现在是 15%。设计本来是居中的，后来同意改成这样了。

**Comment by QingWei-Li** [comment]
冲突啦

**Comment by Leopoldthecoder** [comment]
resolved

File: packages/dialog/src/component.vue (hunk 1 above)
Comment (erguotou520) [inline packages/dialog/src/component.vue]: 居中并不能通过15%来控制吧

**Pull request merged**